import call_functions

print('Success!')
